measurer/base_measurer.go

```diff
@@ -109,9 +114,11 @@ func baseFromMongodbAtlasProcess(p *mongodbatlas.Process) *Base {
 		//We append the port to the UserAlias so that UserAlias becomes unique.
 		//Often the MONGOS is hosted on the same host as the REPLICAS so only
 		//the port will make it unique.
-		UserAlias: p.UserAlias + fmt.Sprintf(":%d", p.Port),
-		TypeName:  p.TypeName,
-		Hostname:  p.Hostname,
-		ID:        p.ID,
+		UserAlias:       p.UserAlias + fmt.Sprintf(":%d", p.Port),
+		TypeName:        strings.Split(p.TypeName, "_")[1],
+		Hostname:        p.Hostname,
+		ID:              p.ID,
+		Sharded_cluster: strings.Split(p.ReplicaSetName, "-")[0] + "-" + strings.Split(p.ReplicaSetName, "-")[1],
+		Project_name:    strings.Split(p.UserAlias, "-")[0] + "-" + strings.Split(p.UserAlias, "-")[1],
```

Comment:
Same here:
Here is some more elegant way to cut the suffix, if it's really necessary:
```suggestion
		Project_name:    strings.Join(strings.Split(p.UserAlias, "-")[:2], "-"),
```